etstool.py

```diff
@@ -125,7 +125,8 @@
 extra_dependencies = {
     # XXX once pyside2 is available in EDM, we will want it here
     'pyside2': set(),
-    'pyqt': {'pyqt5'},
+    'pyqt': {'pyqt<4.12'},  # FIXME: build of 4.12-1 appears to be bad
```

Comment:
4.12.1-1 is available. Do we want to try using that?

Reply:
I don't know... this is the part where I went: "I am not sure if I could do things differently, let's do what we did before".

Reply:
I will try using `4.12.1` in a separate PR and see if it changes test behaviour.

Reply:
yep yep. the changes doesn't have to be in this PR but I just wanted to see if 4.12.1 is any better/safer/less crashy.

Reply:
I just tried 4.12.1, and a test in `TestViewApplication` will segfault. The clean up code looks suspicious.
But the rest of the tests seem to behave well.